apis/types/task_metrics_request.go

```diff
@@ -74,6 +74,10 @@ func (m *TaskMetricsRequest) Validate(formats strfmt.Registry) error {
 		res = append(res, err)
 	}
 
+	if err := m.validateFileLength(formats); err != nil {
```

Comment:
Please do not edit this file directory, you should update swagger.yml first, and then auto-generate this file:
https://github.com/dragonflyoss/Dragonfly/blob/master/apis/README.md